manuscripts/esquery.py

```diff
@@ -64,216 +61,148 @@ def __get_query_filters(cls, filters=None, inverse=False):
                 continue
             field_name = name[1:] if name[0] == '*' else name
-            query_filters += """
-                {
-                  "match": {
-                    "%s": {
-                      "query": "%s",
-                      "type": "phrase"
-                    }
-                  }
-                }
-            """ % (field_name, filters[name])
-            query_filters += ","
 
-        query_filters = query_filters[:-1]  # Remove the last comma
+            params = {field_name: filters[name]}
+            # trying to use elasticsearch_dsl only and not creating hard coded queries
+            query_filters.append(Q('match_phrase', **params))
 
         return query_filters
 
     @classmethod
     def __get_query_range(cls, date_field, start=None, end=None):
         """
-        Create a string with range DSL query for the date in date_field from start to end dates.
+        Create a filter dict with date_field from start to end dates.
 
         :param date_field: field with the date value
         :param start: date with the from value
         :param end: date with the to value
-        :return: a string including the range date DSL query
+        :return: a dict containing a range filter
         """
-
         if not start and not end:
             return ''
 
-        start_end = ''
+        start_end = {}
         if start:
-            start_end = '"gte": "%s",' % start.isoformat()
+            start_end["gte"] = "%s" % start.isoformat()
         if end:
-            start_end += '"lte": "%s",' % end.isoformat()
-        start_end = start_end[:-1]  # remove last comma
-
-        query_range = """
-        {
-          "range": {
-            "%s": {
-              %s
-            }
-          }
-        }
-        """ % (date_field, start_end)
+            start_end["lte"] = "%s" % end.isoformat()
 
+        query_range = {date_field: start_end}
         return query_range
 
     @classmethod
     def __get_query_basic(cls, date_field=None, start=None, end=None,
                           filters=None):
         """
-        Create a string with the date range and filters DSL query.
+        Create a query object with the date range and filters.
 
         :param date_field: field with the date value
         :param start: date with the from value
         :param end: date with the to value
         :param filters: dict with the filters to be applied
-        :return: a string including the DSL query
+        :return: a DSL query containing the required parameters
         """
-        if not date_field:
-            query_range = ''
-        else:
-            query_range = cls.__get_query_range(date_field, start, end)
-            if query_range:
-                query_range = ", " + query_range
+        query_basic = Search()
+
+        # Not using this for now.
+        # query_basic = query_basic.query("query_string", analyse_wildcard="true", query="*")
```

Comment:
Any reason for keeping the comment?

Reply:
No, nothing special. I was just going to experiment with the wildcard flag.

Reply:
In that case, maybe remove it now, and produce later a pr with the extra functionality, if you find it convenient.